modules/administration/pages/custom-channels.adoc

```diff
@@ -0,0 +1,95 @@
+[[custom-channel]]
+= Custom Channels
+
+While {productname} provides all required channels, you might find it useful to create custom channels specific to your environment.
+{productname} Administrators and Channel Administrators have channel management authority, which gives them the ability to create and manage their own custom channels.
+
+Custom channels give you the ability to create your own software packages and repositories, which you can use to update your clients.
```

Comment:
Can you read the sentence before this and this together? For me it sounds like a duplicate at least a part of the sentence. But I am not a native speaker.

Reply:
Yeah, I see what you're saying, but in my defence: The focus of the sentence on L5 is needing admin privs. The focus of the sentence on L7 are the types of things you can do. I'll reword :smile: